README.md

```diff
@@ -1,2 +1,6 @@
-# ci.docker.ibm-http-server
-Dockerfiles for IBM HTTP Server 
+# IBM HTTP Server and Docker
+
+Under this directory you can find build scripts for Docker images (Dockerfiles) and Docker related utilities for IBM HTTP Server 
```

Comment:
Should probably removed the part after the 'and' as there are no other related utilities in this case.